<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment></comment>
 <class>CtrlPanel</class>
 <widget class="QWidget" name="CtrlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <property name="whatsThis">
   <string>Choose the item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>3</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="contentPage">
      <property name="whatsThis">
       <string/>
      </property>
      <attribute name="title">
       <string>Workspace</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTreeWidget" name="treeContents">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::NoDragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::IgnoreAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="autoExpandDelay">
           <number>2</number>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="searchPage">
      <attribute name="title">
       <string>Search</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="3" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="TextLabel1_2">
         <property name="text">
          <string>Searching f&amp;or:</string>
         </property>
         <property name="buddy">
          <cstring>termsEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="termsEdit">
         <property name="toolTip">
          <string>Enter searchword(s).</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QListWidget" name="resultBox">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="TextLabel2">
         <property name="text">
          <string>Found Objects:</string>
         </property>
         <property name="buddy">
          <cstring>resultBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="helpButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Display the help page.</string>
           </property>
           <property name="whatsThis">
            <string>Display the help page for the full text search.</string>
           </property>
           <property name="text">
            <string>He&amp;lp</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>61</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Start searching.</string>
           </property>
           <property name="whatsThis">
            <string>Pressing this button starts the search.</string>
           </property>
           <property name="text">
            <string>&amp;Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="indexPage">
      <attribute name="title">
       <string>2</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="bookmarkPage">
      <attribute name="title">
       <string>3</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="listBookmarks">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="whatsThis">
          <string>Displays the list of bookmarks.</string>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>column 1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonAdd">
           <property name="toolTip">
            <string>Add new bookmark</string>
           </property>
           <property name="whatsThis">
            <string>Add the currently displayed page as a new bookmark.</string>
           </property>
           <property name="text">
            <string>&amp;New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRemove">
           <property name="toolTip">
            <string>Delete bookmark</string>
           </property>
           <property name="whatsThis">
            <string>Delete the selected bookmark.</string>
           </property>
           <property name="text">
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>treeContents</tabstop>
  <tabstop>listBookmarks</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>buttonRemove</tabstop>
  <tabstop>termsEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>resultBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
